components/txn_types/src/types.rs

```diff
@@ -316,17 +322,34 @@ impl From<kvrpcpb::Mutation> for Mutation {
     }
 }
 
-#[derive(Default, Debug, Clone, PartialEq)]
-pub struct OldValue {
-    pub short_value: Option<Value>,
-    pub start_ts: TimeStamp,
+#[derive(Debug, Clone, PartialEq)]
+pub enum OldValue {
+    Value {
+        short_value: Option<Value>,
+        start_ts: TimeStamp,
+    },
+    NotExist,
```

Comment:
What is the difference between `Some(NotExist)` and `None`? Perhaps we can add another variant to the enum with a more descriptive name.

Please add some docs to the enum and variants.

Reply:
See discussion [here](https://github.com/tikv/tikv/pull/9173#discussion_r534277441).
So maybe we can add a new variant `DontCare`?

Reply:
Yeah, I think having a three-valued enum is a good idea. Maybe `Value`, `None`, and `Unspecified`?